frontend/src/ui/Blocks/Series.tsx

```diff
@@ -373,6 +716,7 @@ const GridTile: React.FC<GridTypeProps> = ({ event, basePath, active }) => {
         },
         ...active && {
             backgroundColor: "var(--grey86)",
+            display: "inline-block",
```

Comment:
Can you explain why this change (and the corresponding one below) was necessary?

Reply:
Errm it is not. I think I just wondered why this wasn't in here but in the return statement instead, so I played around with it and forgot to change it back.